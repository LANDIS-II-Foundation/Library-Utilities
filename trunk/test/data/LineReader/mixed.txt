# This file has blank lines, comment lines and EOL comments.
# Some of the comment lines, like these first 3 lines, use
# the default comment-line marker "#".

//	These lines use a different comment-line marker, which is
//	the C++/C# comment marker.

Here's an EOL comment with default marker -->  ## (EOL comment)

################# Another EOL comment
   	   	  
The line above has just whitespace.

  These next lines have EOL comments which use		// a EOL comment
  a different marker than the default marker;		/// Another EOL comment
  they use the C++ comment marker.

# Comment line with default marker, but with   ## default EOL comment
# Comment line with default marker, but with   // alternate EOL comment
//Comment line with alternate marker, but with ## default EOL comment
//Comment line with alternate marker, but with // alternate EOL comment
