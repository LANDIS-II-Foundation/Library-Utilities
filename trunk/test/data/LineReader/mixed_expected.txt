1:# This file has blank lines, comment lines and EOL comments.
2:# Some of the comment lines, like these first 3 lines, use
3:# the default comment-line marker "#".
4:
5://	These lines use a different comment-line marker, which is
6://	the C++/C# comment marker.
7:
8:Here's an EOL comment with default marker -->  ## (EOL comment)
9:
10:################# Another EOL comment
11:   	   	  
12:The line above has just whitespace.
13:
14:  These next lines have EOL comments which use		// a EOL comment
15:  a different marker than the default marker;		/// Another EOL comment
16:  they use the C++ comment marker.
17:
18:# Comment line with default marker, but with   ## default EOL comment
19:# Comment line with default marker, but with   // alternate EOL comment
20://Comment line with alternate marker, but with ## default EOL comment
21://Comment line with alternate marker, but with // alternate EOL comment
